README.md

```diff
@@ -27,7 +27,10 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 
 Write your code here:
 ```ruby
-# code here
+def offer_rose person
+  puts "Would you take this rose, #{person}, in exchange for giving an old beggar woman shelter from
+  the bitter cold?"
+
```

Comment:
missing the `end` keyword here to close out the method, and need to call the method i.e. 
```ruby
offer_rose "young_prince"
```